### ISSUE TITLE
Ipums calculations and Table 2B

### ISSUE THREAD

**Issue by rpitingolo** [NeighborhoodInfoDC/DMPED#6]
Using iPums data for ACS 2012-16 pull in variables and calculate indicators for households with 4+ people.

**Comment by lhendey** [comment]
@rpitingolo @maggiesu0725 - this file https://github.com/NeighborhoodInfoDC/Equity/blob/master/Prog/IPUMS_2010_14.sas  has some code for race, etc. that could be useful. please double check it.

**Comment by rpitingolo** [comment]
@lhendey  thanks. @maggiesu0725 asked me about calculating the income categories. Is it OK to use if/then statements to code this table into the program: https://www.huduser.gov/portal/datasets/il/il2016/2016summary.odn Or should she start with the Median income number ($108,600) and then calculate based on the formula in this doc: https://code.dccouncil.us/dc/council/code/titles/42/chapters/28/ ?

**Comment by lhendey** [comment]
@rpitingolo @maggiesu0725 - probably helpful to have both versions.

**Comment by lhendey** [comment]
@rpitingolo - have you seen any code on this one? I didn't see a pull request.  

Please add employment status and educational attainment.  Note that we need cost burden by tenure too. 

DO you think this piece goes in a separate issue? 
What are the characteristics of affordable family sized rental units? (from IPUMS)
o	Range of bedroom sizes
o	Total number of units in structure
o	Age of structure
o	Vacancy status

**Comment by rpitingolo** [comment]
@lhendey I'm adding these in now.

**Comment by rpitingolo** [comment]
@lhendey right now we're defining large unit by number of people in the HH. For the vacants units we obviously don't have that. Do you want me to use bedroom size? That would let me create the tables you want. The downside is the inconsistent definition.

**Comment by lhendey** [comment]
@rpitingolo - can you finish this today? i'm out next week.

**Comment by rpitingolo** [comment]
@lhendey yeah. I may need to work on it tonight. I'm still trying to figure out the right formula for the over/under.

**Comment by ptatian** [comment]
@rpitingolo Leah asked me to take a look at the PUMA to Ward issue for the IPUMS data. Should I use the code you have here as a starting point?

**Comment by rpitingolo** [comment]
@ptatian this is a good place to start. You'll see some proc summaries where we have `class puma largehh`. I think this is where we'll put the ward variable we create. Let me know if you have any questions.
